app/views/user_mailer/verify_email.html.erb

```diff
@@ -0,0 +1,35 @@
+<%
+# BigBlueButton open source conferencing system - http://www.bigbluebutton.org/.
+#
+# Copyright (c) 2018 BigBlueButton Inc. and by respective authors (see below).
+#
+# This program is free software; you can redistribute it and/or modify it under the
+# terms of the GNU Lesser General Public License as published by the Free Software
+# Foundation; either version 3.0 of the License, or (at your option) any later
+# version.
+#
+# BigBlueButton is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License along
+# with BigBlueButton; if not, see <http://www.gnu.org/licenses/>.
+%>
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
+  </head>
+  <body>
+    <h1>Welcome to Greenlight!, <%= @user.name %></h1>
+    <p>
+      You have successfully signed up for Greenlight,
+      your username is: <%= @user.email %>.<br>
+    </p>
+    <p>
+      To verify your account, just follow this link: <%= @url %>.
```

Comment:
The link should be an actual link. It seems that is not working. Here the generated html code

```
<html>
  <head>
    <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
  </head>
  <body>
    <h1>Welcome to Greenlight!, Juan Pueblo</h1>
    <p>
      You have successfully signed up for Greenlight,
      your username is: jpueblo@mailinator.net.<br>
    </p>
    <p>
      To verify your account, just follow this link: http://rails-dev:3002/u/verify/confirm/gl-vkqwsrucnnbm.
    </p>
    <p>Thanks for joining and have a great day!</p>
  </body>
</html>
  </body>
</html>
```